### PR TITLE
Armadillo controller including autoload.

### PR DESCRIPTION
**Pull request by clemens-tolboom**
This is my take for an Armadillo controller to control through the Armadillo API the docker containers.

We can `start`, `stop`, `restart`, `status` with `*All` and `autoStart`.

Using `autoStart` the settings from the environment can be used.

### CODE REVIEW

File: scripts/ops/armadilloctl.zsh [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+# FIXME: loads from local files
+if [[ -f ".env" ]]
+then
+  echo "Sourcing .env"
+  source ".env"
+else
+  echo "Sourcing dev.env"
+  source "dev.env"
+fi
```
Comment (clemens-tolboom) [inline scripts/ops/armadilloctl.zsh:163]: I'm not happy with this code as the file location of the `.env` should be discoverable. Feedback is welcome

Comment (clemens-tolboom) [reply scripts/ops/armadilloctl.zsh:163]: I think about not including the `.env` but just expect that the environment vars are set when running.

Comment (clemens-tolboom) [reply scripts/ops/armadilloctl.zsh]: Removed and documentation added. Ready for testing.

**Comment by marikaris** [comment]
[Armadillo controller including autoload](https://trello.com/c/tQxzZTG9/1110-armadillo-controller-including-autoload)

File: scripts/ops/env.dist [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# copy this file to ie "/etc/armadillo/acc.env"
```
Comment (marikaris) [inline scripts/ops/env.dist:1]: Why do you suggest another path than in the `.README`? I suggest keeping it the same.

**Review by marikaris — COMMENTED** [review]
Do we maybe want to add the .env files to the `.gitignore`? I'd suggest adding `scrips/ops/conf/armadillo.acc.env` to `.gitignore` (this goes for `scripts/release/.env` as well, so if you agree, maybe you could add this one as well)

File: scripts/ops/env.dist [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+# FIXME: We could / should test for admin user in overload config file instead?
```
[the remark this answers is not in the document]
Comment (clemens-tolboom) [reply scripts/ops/env.dist:3]: That's a weird sentence ... when installing Armadillo admin user / password is already in `application.yml`. I was somehow referring the that file :-O

File: scripts/ops/env.dist (hunk 2 above)
Comment (clemens-tolboom) [reply scripts/ops/env.dist:1]: Thanks ... done

**Comment by clemens-tolboom** [comment]
Set to WIP as it should be `bash` instead of `zsh`

File: scripts/ops/env.dist (hunk 3 above)
[the remark this answers is not in the document]
Comment (clemens-tolboom) [reply scripts/ops/env.dist]: I've removed it for now. Let first move this to old environments to test :)

File: scripts/ops/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Armadillo for OPS
+
+To monitor and control Armadillo and its DataShield docker containers we need to use the Armadillo API.
+
+## Armadillo Controller
+
+With `armadilloctl` you can control the DataShield docker containers.
```
Comment (mswertz) [inline scripts/ops/README.md:7]: who is the envisioned user for this? I mean why is this outside of the app?

Comment (clemens-tolboom) [reply scripts/ops/README.md:7]: From hindsight a checkbox in Armadillo was probably enough ... the code was already written as investigation/spinoff from the Rock only PR.

**Review by marikaris — APPROVED** [review]

**Pull request merged**